src/replacer.js

```diff
@@ -47,31 +46,52 @@ var insert_symbols = new function() {
         }
     }
 
-    this.onKeyUp = function(evt) {
+    this.onKeyUp = function (evt) {
         if (shouldCheckOnKeyup) {
             shouldCheckOnKeyup = false;
             checkForReplacement(evt.currentTarget.getRootNode(), false);
         }
     }
-
     /**
      * Add event handlers to Editor key events. Setup only needs to be 
      * performed when the editor is first created.
      */
-    forEditorField([], (field) => {
-        if (!field.hasAttribute("has-type-symbols")) {
-            field.editingArea.editable.addEventListener("keydown", this.onKeyDown)
-            field.editingArea.editable.addEventListener("keyup", this.onKeyUp)
-            field.setAttribute("has-type-symbols", "")
-        }
-    })
+    if (typeof forEditorField !== 'undefined') {
+        // for Anki < 2.1.50
+        forEditorField([], (field) => {
+            if (!field.hasAttribute("has-type-symbols")) {
+                field.editingArea.editable.addEventListener("keydown", this.onKeyDown)
+                field.editingArea.editable.addEventListener("keyup", this.onKeyUp)
+                field.setAttribute("has-type-symbols", "")
+            }
+        })
+    } else {
+        // for Anki >= 2.1.50
+        setTimeout(() => {
+            require("anki/ui").loaded.then(() => {
+                fields = require("anki/NoteEditor").instances[0].fields
+                for (const field of fields) {
+                    field.element.then((fieldElm) => {
+                        if (!fieldElm.hasAttribute("has-type-symbols")) {
+                            const editingArea = fieldElm.getElementsByClassName("editing-area")[0];
+                            const shadowRoot = editingArea.getElementsByClassName("rich-text-editable")[0].shadowRoot;
+                            const editable = shadowRoot?.querySelector("anki-editable");
```

Comment:
Should the '?' be there?

Reply:
It works like that for me. Does it not work for you? I think the "?." syntax works only on some platforms. I'll change it.

Reply:
Thanks! When I ran it, it wouldn't work until I removed the '?'

Reply:
Good to know. Thanks!